system/mocks/mocks.go

```diff
@@ -6,290 +6,289 @@ package mocks
 
 import (
 	context "context"
-	reflect "reflect"
-
 	gomock "github.com/golang/mock/gomock"
 	types "github.com/spacemeshos/go-spacemesh/common/types"
+	reflect "reflect"
 )
 
-// MockFetcher is a mock of Fetcher interface.
+// MockFetcher is a mock of Fetcher interface
```

Comment:
do you know why this generated code got changed?

Reply:
I changed a signature of one of the mocked methods, re-generated mocks, then reverted the change and re-generated again